docs/tutorials/09-complete-tutorials/05-smart-contract.md

```diff
@@ -5,34 +5,28 @@ title: Smart Contract Verification
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-In this tutorial we will be developing a smart contract which verifies the attestation proof by calling the zkVerify contract. We will be using Remix toolkit to develop the smart contract which will have the interface to verify our attestations. Also we have a [foundry template](https://github.com/zkVerify/zkverify-evm-dapp-example) you can check out.
+In this tutorial we will be developing a smart contract which verifies the aggregation proof by calling the zkVerify contract. We will be using Remix toolkit to develop the smart contract which will have the interface to verify our aggregations. Also we have a [foundry template](https://github.com/zkVerify/zkverify-evm-dapp-example) you can check out.
 
-First we need to create an interface to interact with zkVerify smart contracts. Create a new file called as IZKVerify.sol and use the following code snippet to create the interface :
+First we need to create an interface to interact with zkVerify smart contracts. Create a new file called as ``IVerifyProofAggregation.sol`` and use the following code snippet to create the interface :
 
 ```solidity
-// SPDX-License-Identifier: UNLICENSED
-pragma solidity ^0.8.13;
+// SPDX-License-Identifier: Apache-2.0
 
-interface IZkVerifyAttestation {
+pragma solidity 0.8.20;
 
-    function submitAttestation(
-        uint256 _attestationId,
-        bytes32 _proofsAttestation) external;
-
-    function submitAttestationBatch(
-        uint256[] calldata _attestationIds,
-        bytes32[] calldata _proofsAttestation) external;
-
-    function verifyProofAttestation(
-        uint256 _attestationId,
+interface IVerifyProofAggregation {
+    function verifyProofAggregation(
+        uint256 _domainId,
+        uint256 _aggregationId,
         bytes32 _leaf,
         bytes32[] calldata _merklePath,
         uint256 _leafCount,
-        uint256 _index) external returns (bool);
+        uint256 _index
+    ) external view returns (bool);
 }
 ```
 
-After this, we are ready to create our main contract which will check attestations and can perform any function as per the business logic. To get started first we declare our proving system which can be different for each proof type, this will be used while generating the leaf digest to verify the attestations.
+After this, we are ready to create our main contract which will verify aggregation proofs and can perform any function as per the business logic. To get started first we declare our proving system which can be different for each proof type, this will be used while generating the leaf digest to verify the aggregation.
```

Comment:
It's worth mentioning in this tutorial also the utility libraries that Luca G. developed https://github.com/zkVerify/zkv-attestation-contracts/tree/main/contracts/verifiers